lang_python.adoc

```diff
@@ -17,21 +17,29 @@ To generate Python modules, Kaitai Struct compiler should be invoked with
 kaitai-struct-compiler -t python my_spec.ksy
 
 This normally generates `my_spec.py` parser module, which is compatible
-with both Python 2 and 3.
+with both Python 2 and 3. It is strongly recommended to use python 3
+because python 2 is EOL since 2020-01-01.
 
-TODO: document `--python-package`
+`--python-package` allows you to select a parent package name used by
+one generated file to import symbols from another one. If you want to
+embed KSC-generated files into an own package, it is recommended to
+set it to `.`, which means "current package" rather than to the real name
+of the subpackage, because setting it to a real name will cause breakage
+when you want to reorganize or rename your package. 
```

Comment:
First information should be what the `--python-package` is at all and how one can use it, before you tell what it _allows_. It should be mentioned here that it's a command line option that one can supply to the `kaitai-struct-compiler` command, and also some common usage examples would come in handy. It can be also already present as "optional parameter" in the first command like `kaitai-struct-compiler -t python my_spec.ksy [ --python-package whatever ]` so it's clear.

Reply:
It would be also cool to show what it actually translates to in the generated Python code. For example, you can show that passing `--python-package ...` causes insertion of line `package ...` (don't actually know, just guessing).

Reply:
Done.